api/src/query.rs

```diff
@@ -207,7 +207,7 @@ impl QueryRoot {
             .map(|agg| Account {
                 keyid: agg.keyid,
                 chain_key: agg.chain_key,
-                packet_key: agg.packet_key,
+                packet_key: format!("0x{}", agg.packet_key),
```

Comment:
I guess it is not needed here, as parsing can usually distinguish if prefix is present or not. 

Moreover, what if the prefix is already present?